resources/generator.php

```diff
@@ -1,130 +1,152 @@
-        if($file_upload){
-            print "<img src='".$path."'>";
-        }else{
-            print "no hay foto";
+        if($current_job == "on"){
+            $date_job_end = date("Y-m-d");
         }
-    }else{
-        print "No se a podido realizar la inserccion por: <br>".$bugs;
-    }
 
-    switch ($design) {
-        case "1":
-          print "serio<br>";
-            break;
-        case "2":
-            print "estrobertido<br>";
-            break;
-        case "3":
-            print "dinamico<br>";
-            break;
-        default:
-            print "petardazo";
-            break;
-    }
-    
-    print ($first_name."<br>".$last_name."<br>");
-    print ($home.", ".$city.", ".$country.", ".$postal_code."<br>");
-    print ($type_phone.": ".$number_phone."<br>");
-    print ($email."<br>");
+        @$num_licenses = count($drive_license);
+        $have_license = false;
+        $licenses = "";
+        if($num_licenses != 0){
+            $have_license = true;
 
-    if(!(empty($job))){
-        print ($job."<br>");
-        print ($employer_name."<br>");
-        print ($employer_city."<br>");
-        print ($employer_country."<br>");
-        print ($date_job_start."<br>");
-        if($current_job == "on"){
-            $d=strtotime("today");
-            echo "trabajando a ".date("Y-m-d", $d)."<br>";
-        }else{
-            print ($date_job_end."<br>");
+            $licenses = $licenses.$drive_license[0];
+            for($i=1;$i<$num_licenses;$i++){
+                $licenses = $licenses.", ".$drive_license[$i];
+            }
         }
-    }else{
-        print("sin esperiencia laboral<br>");
-    }
-    
-    if(!(empty($title))){
-        print ($title."<br>");
-        print ($school_name."<br>");
-        print ($school_city."<br>");
-        print ($school_country."<br>");
-        print ($acquired_skills."<br>");
-        print ($date_education_start."<br>");
-        if($current_education == "on"){
-            $d=strtotime("today");
-            echo date("Y-m-d", $d) . "<br>";
+
+        $bugs = "";
+        $file_upload = false;
+        $path = "";
+        if (is_uploaded_file ($_FILES['picture']['tmp_name'])){
+            $folder = "../src/img/users/";
+            $name_file = $_FILES['picture']['name'];
+            $file_upload = true;
+            $path = $folder.$name_file;
+            if (is_file($path)){
+                $name_file = time()."_".$name_file;
+            }
+            $path = $folder.$name_file;
+        }else if($_FILES['picture']['error'] == UPLOAD_ERR_FORM_SIZE){
+            $maximun_file_size = $_POST['maximun_file_size'];
+            $bugs = $bugs."No se puede subir una imagen tan pesada, el maximo es ".$maximun_file_size." bytes<br>";
+        }else if($_FILES['picture']['name'] == ""){
+            $name_file = '';
         }else{
-            print ($date_education_end."<br>");
+            $bugs = $bugs."No se pudo subir la imagen.<br>";
```

Comment:
Salvo los valores que se recogen, el resto de cadenas se deben de traducir usando [gettext](https://www.php.net/manual/es/book.gettext.php). Ya que cogí como msgid el idioma inglés, para evitar confusión, todas la cadenas deben estar en inglés.

Por ejemplo en esta línea `$bugs = $bugs._("Could not upload image.<br>)";`

Las cadenas van así `_("Traslate me!")`, con eso Poedit las reconoce y podemos generar los Po de otros idiomas